leonardo-lfonzi62/week-3/giffaw-master/scripts/app.js

```diff
@@ -0,0 +1,35 @@
+var giphyUrl = "http://api.giphy.com/v1/gifs/search?q="
+var key = "55wBK7VLZy5IoUOK0dFBERd0vHRdaN8A";
+
+$('form').on('submit', function(e) {
+  e.preventDefault();
+  // emptys out the div where the previous gifs on the page were located
+ $(`.gif-gallery`).empty();
+var searchTerm = $(`.form-control`).val();
+
+  $.ajax({
+    method: "GET",
+    url: giphyUrl + searchTerm + "&api_key=" + key + "&offset=5",
```

Comment:
Nice url breakdown. The only issue I see is that you do not need to put a key in your js because it is already in your form submission. It is good practice to go through all code given to you to keep yourself from extra work and repetitive code.